README.md

```diff
@@ -1,2 +1,27 @@
 # politico
 Politico- A web application that enables citizens give their mandate to politicians running for different government offices while building trust in the process through transparency.
+
+# Features
+* A User can:
+  * create an account
+  * login into the application
```

Comment:
I thought a user can do more than the actions listed above

Reply:
Yes, a user can do more than that but I only included the features I have currently developed a UI for.

Reply:
Ok, that means you'll be updating the README as the codebase progresses right?

Reply:
Yes I will.